### PR TITLE
fix: Pinned protobuf to prevent issues with pb files.

### PR DESCRIPTION
**Pull request by ivanmkc**
Currently, we rely on dependencies like proto-plus-python to select a protobuf library version.

However, if they select an incompatible version, it can lead to issues like https://github.com/googleapis/python-aiplatform/issues/1257

Additionally, proto-plus and packaging had upper version limits added to prevent future breaking changes.

Prohibited versions were chosen from https://pypi.org/project/packaging/#history and https://pypi.org/project/proto-plus/#history and represent the next major release.

### CODE REVIEW

**Review by parthea — APPROVED** [review]

**Comment by parthea** [comment]
@sararob, Please could you review/approve?

File: setup.py [hunk 1]
```diff
@@ -103,8 +103,9 @@
         # Until this issue is closed
         # https://github.com/googleapis/google-cloud-python/issues/10566
         "google-api-core[grpc] >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
-        "proto-plus >= 1.15.0",
-        "packaging >= 14.3",
+        "proto-plus >= 1.15.0, <2.0.0dev",
+        "protobuf >= 3.19.0, <4.0.0dev",
```
Comment (parthea) [inline setup.py:107]: Please also add `protobuf==3.19.0` to test the minimum supported version in `testing/constraints-3.6.txt`:

https://github.com/googleapis/python-aiplatform/blob/0036ab07004e0c9ae7806c4c2c25f22d5af4a978/testing/constraints-3.6.txt#L1-L8

Comment (ivanmkc) [reply setup.py:107]: Thanks, added.

**Comment by sararob** [comment]
LGTM once we confirm system tests pass.

**Review by samgoodman — APPROVED** [review]

**Review by samgoodman — APPROVED** [review]

**Pull request merged**